lib/smart_answer_flows/locales/en/overseas-passports.yml

```diff
@@ -454,29 +448,29 @@ en-GB:
         how_long_6_weeks: |
-          Your application will take **at least** 6 weeks from when it’s received by Her Majesty's Passport Office in the UK.
+          Your application should take 6 weeks from when it’s received by Her Majesty’s Passport Office in the UK.
         how_long_8_weeks: |
-          Your application will take **at least** 8 weeks from when it’s received by Her Majesty's Passport Office in the UK.
+          Your application should take 8 weeks from when it’s received by Her Majesty’s Passport Office in the UK.
         how_long_10_weeks: |
-          Your application will take **at least** 10 weeks from when it’s received by Her Majesty's Passport Office in the UK.
+          Your application should take 10 weeks from when it’s received by Her Majesty’s Passport Office in the UK.
         how_long_12_weeks: |
-          Your application will take **at least** 12 weeks from when it’s received by Her Majesty's Passport Office in the UK.
+          Your application should take 12 weeks from when it’s received by Her Majesty’s Passport Office in the UK.
         how_long_14_weeks: |
-          Your application will take **at least** 14 weeks from when it’s received by Her Majesty's Passport Office in the UK.
+          Your application should take 14 weeks from when it’s received by Her Majesty’s Passport Office in the UK.
         how_long_16_weeks: |
-          Your application will take **at least** 16 weeks from when it’s received by Her Majesty's Passport Office in the UK.
+          Your application should take 16 weeks from when it’s received by Her Majesty’s Passport Office in the UK.
         how_long_18_weeks: |
-          Your application will take **at least** 18 weeks from when it’s received by Her Majesty's Passport Office in the UK.
+          Your application should take 18 weeks from when it’s received by Her Majesty’s Passport Office in the UK.
         how_long_5_months: |
-          Your application will take **at least** 5 months from when it’s received by Her Majesty's Passport Office in the UK. You may have to attend an interview.
+          Your application should take 5 months from when it’s received by Her Majesty’s Passport Office in the UK. You may have to attend an interview.
         how_long_6_months: |
-          Your application will take **at least** 6 months from when it’s received by Her Majesty's Passport Office in the UK. You may have to attend an interview.
+          Your application should take 6 months from when it’s received by Her Majesty’s Passport Office in the UK. You may have to attend an interview.
         how_long_8_months: |
-          Your application will take **at least** 8 months from when it’s received by Her Majesty's Passport Office in the UK. You may have to attend an interview.
+          Your application should take 8 months from when it’s received by Her Majesty’s Passport Office in the UK. You may have to attend an interview.
         report_loss_or_theft: |
```

Comment:
We decided not to refactor, because the copy for weeks and months differs and introduced complexity would not make the situation better.